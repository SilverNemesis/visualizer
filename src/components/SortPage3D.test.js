import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import SortPage3D from './SortPage3D'

const renderer = new ShallowRenderer();

it('renders correctly', () => {
  renderer.render(<SortPage3D />)
  const tree = renderer.getRenderOutput()
  expect(tree).toMatchSnapshot()
});