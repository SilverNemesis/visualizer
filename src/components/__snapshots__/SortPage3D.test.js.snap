// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Section
  inner={true}
>
  <Container
    className="mt-2 mb-2"
    fluid={true}
  >
    <Row
      row="sm"
    >
      <Col
        className="d-flex justify-content-around align-items-center"
        col="sm"
      >
        <Button
          disabled={false}
          onClick={[Function]}
          styles="primary"
        >
          Shuffle
        </Button>
        <Button
          disabled={false}
          onClick={[Function]}
          styles="primary"
        >
          Reverse
        </Button>
        <Button
          disabled={false}
          onClick={[Function]}
          styles="primary"
        >
          Bubble Sort
        </Button>
        <Button
          disabled={false}
          onClick={[Function]}
          styles="primary"
        >
          Insertion Sort
        </Button>
        <Button
          disabled={false}
          onClick={[Function]}
          styles="primary"
        >
          Merge Sort (In Place)
        </Button>
        <Button
          disabled={false}
          onClick={[Function]}
          styles="primary"
        >
          Merge Sort
        </Button>
        <Button
          disabled={false}
          onClick={[Function]}
          styles="primary"
        >
          Quick Sort
        </Button>
      </Col>
    </Row>
  </Container>
  <canvas
    className="canvas"
    onClick={[Function]}
  />
</Section>
`;
